xbmc/pvr/channels/PVRChannelGroup.h

```diff
@@ -260,9 +260,10 @@ namespace PVR
 
     /*!
      * @brief Remove invalid channels and updates the channel numbers.
+     * @param bIgnoreGroupChannnelNumbersFromOne if True don't number channels starting from one.
      * @return True if something changed, false otherwise.
      */
-    bool Renumber();
+    bool Renumber(bool bIgnoreGroupChannnelNumbersFromOne = false);
```

Comment:
Any chance not to introduce a bool param that nobody can understand when reading the calling code ("Hmm, what will this "false" in "Renumber(false);" do? Damn, must loook it up in the declaration/implementation to understand.")?

Reply:
Ya, that was the original intention. The other is use a separate function to do this intial renumber (which is what I had prior to this). Let me go back to that.

Reply:
Another option would be to use an enum so intention is clear.

Reply:
enum would be fine, yes.